@here/harp-datasource-protocol/test/ExprEvaluatorTest.ts

```diff
@@ -1584,16 +1667,6 @@ describe("ExprEvaluator", function() {
                 ["step", ["zoom"], 0, 2, 123]
             );
 
```

Comment:
Accidental ?

Reply:
what so you mean?

Reply:
Sorry. Totally bogus comment i misread patch completly. Never mind.